tests/copp/copp_utils.py

```diff
@@ -37,6 +39,18 @@ def limit_policer(dut, pps_limit):
     dut.script(cmd="{} {} {} {}".format(_UPDATE_COPP_SCRIPT, pps_limit,
                                         _BASE_COPP_CONFIG, _TEMP_COPP_CONFIG))
     dut.command("docker cp {} {}".format(_TEMP_COPP_CONFIG, _SWSS_COPP_CONFIG))
+    # Comment out copp config generation in start.sh
+    copp_gen_script_text = '#!/bin/bash\n' + \
+                    'docker exec -ti swss bash -c ' + \
+                    '\"sed -i \'s+sonic-cfggen -d -t /usr/share/sonic/templates/copp.json.j2+' + \
+                    '#sonic-cfggen -d -t /usr/share/sonic/templates/copp.json.j2+g\' /usr/bin/start.sh\"'
```

Comment:
Can this be a script that is called in a similar fashion to `_UPDATE_COPP_SCRIPT`?

Reply:
So, you prefer to add it to the repo rather then generate it on the fly, right? Sure, will do this.

Reply:
Yes, I'm already working on this. The problematic point is that the template file doesn't exist  on the older images so it's hard to make the fix backward compatible.

Reply:
> I am also fine. Though I don't understand what is backward compatible image here means ? Both master and 201911 has template change.

@abdosi  The older 201911 builds don't have the copp template. But if this is not so important I can remove this code.

Reply:
I think we need not maintain backward compatibility with older 201911 image. This should be good going forward.

Reply:
@abdosi Removed. Please check now.